.github/workflows/build.yaml

```diff
@@ -67,13 +67,15 @@ jobs:
           java-version: '17.0.4+8'
           distribution: 'adopt'
 
-      - name: Initialize CodeQL
-        uses: github/codeql-action/init@v2
+      - if: "!contains(github.ref, 'dependabot')"
```

Comment:
Copied this from https://github.com/dockstore/dockstore-cli/blob/b1c1300e3c6af0fcc1289b817795cf26c6294265/.github/workflows/mvnw.yml#L34

dependabot only has read-access, but uploading Code Scanning results requires write access. This line avoids running CodeQL if the branch is dependabot's